src/main/java/factionsplusplus/models/PollOption.java

```diff
@@ -0,0 +1,59 @@
+package factionsplusplus.models;
```

Comment:
I wouldn't add dependencies to subclasses, I would just make the top most layer (Poll) have whatever dependencies and handle updates for the lower classes

Reply:
Are you suggesting to remove PollVote and PollOption models?